### ISSUE TITLE
 Optional bottom bar style toggle

### ISSUE THREAD

**Issue by WarnError** [tachiyomiorg/tachiyomi#6903]
### Describe your suggested feature

The bottom bar of the new version looks abrupt under non-Android 12,hope to be able to add options to the settings for switching.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.13.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
No, this aligns with MD3 and is already used in various other apps.
